bus-mapping/src/error.rs

```diff
@@ -38,6 +38,8 @@ pub enum Error {
     WordToMemAddr,
     /// Error while generating a trace.
     TracingError,
+    /// Error while adding EvmWord
+    EvmWordAddingOverflow,
```

Comment:
As said by @ed255 and you can see in the specs, this error is not needed. What we should add is a `StackUnderflow` variant which will get triggered when we call `ADD` but we have less than 2 items in the Stack.

Reply:
Exactly.
I changed.